sunpy/map/maputils.py

```diff
@@ -581,6 +586,9 @@ def pixelate_coord_path(smap, coord_path, *, bresenham=False):
     If a pixel intersects the coordinate path at only its corner, it may not be
     returned due to the limitations of floating-point comparisons.
 
+    If the coordinate path exceeds the bounds of the map, those returned pixel
```

Comment:
this sentence is a bit confusing especially if someone was not familiar with what is going on here. Can it be more explicit? for example, saying that it will correspond to pixels that lie outside the bounds of the passed map etc.

Reply:
I have expanded this note a bunch